### PR TITLE
Devcon fixes

### PR DESCRIPTION
**Pull request by pi0neerpat**

### CODE REVIEW

File: src/dashboard/Tribute.js [hunk 1]
```diff
@@ -1,202 +1,296 @@
...
+    // what happens when approval works but mint gets dropped
+    await this.DAIContract.approve(
+      rDAIContract.address,
+      bigNumberify(amount).mul(WeiPerEther)
+    );
+    // dont call newhat
+    await this.rDAIContract.mint(bigNumberify(amount).mul(WeiPerEther));
+    let output = await this.rDAIContract.getHatByAddress(this.address);
+    console.log(output);
+  };
```
[the remark this answers is not in the document]
Comment (pi0neerpat) [reply src/dashboard/Tribute.js]: We needed a way to generate Tribute. It works on Desktop, but the second tx sometimes never appears on mobile

File: src/dashboard/general/Scanner/Scanner.js [hunk 2]
```diff
@@ -1,310 +1,306 @@
...
+        </div>
+        {displayedImage}
+        {failMessage}
+        {loader}
+      </div>
+    );
+  }
+}
+
+export default Scanner;
```
[the remark this answers is not in the document]
Comment (pi0neerpat) [reply src/dashboard/general/Scanner/Scanner.js]: Its copied from the Burner Wallet.

File: .gitignore [hunk 3]
```diff
@@ -1,7 +1,6 @@
 .cache/
 dist/
 node_modules/
-src/contracts/*
```
[the remark this answers is not in the document]
Comment (ejwessel) [reply .gitignore]: Resolved in https://github.com/Send-Tribute/tribute-rDAI/pull/24

File: package.json [hunk 4]
```diff
@@ -17,14 +17,15 @@
   "scripts": {
     "start": "parcel public/dashboard.html public/widget.html",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "parcel build public/dashboard.html public/widget.html --out-dir dist",
+    "build": "parcel build public/dashboard.html --out-dir dist",
```
[the remark this answers is not in the document]
Comment (ejwessel) [reply package.json]: https://github.com/Send-Tribute/tribute-rDAI/pull/25

**Pull request merged**